README.md

```diff
@@ -98,40 +74,44 @@ configs available in `.ci_support`, for instance:
 CONFIG=linux_64_cuda_compiler_version10.2 .scripts/run_docker_build.sh
 ```
 
-### 2. using cmake directly in a conda environment
+### 2. using cmake directly
+
+This package depends on boost, libtiff, fftw, and cuda.
 
 Here we create a dedicated conda environment with all of the build dependencies
 installed, and then use cmake directly.  This method is faster and creates an
 immediately useable binary (i.e. it is better for iteration if you're changing
 the source code), but requires that you set up build dependencies correctly.
 
-1. install [miniconda](https://docs.conda.io/en/latest/miniconda.html)
-2. install [cudatoolkit](https://developer.nvidia.com/cuda-10.1-download-archive-update2) (I haven't yet tried 10.2)
-3. (*windows only*) install [build tools for VisualStudio](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2017) 2017.  For linux, all necessary build tools will be installed by conda.
+1. install conda ([miniconda](https://docs.conda.io/en/latest/miniconda.html) or [miniforge](https://github.com/conda-forge/miniforge))
+1. (*windows only*) install [build tools for
+   VisualStudio](https://visualstudio.microsoft.com/downloads)
+   2019.  For linux, all necessary build tools will be installed by conda.
 
-4. create a new conda environment with all of the dependencies installed
+1. create a new conda environment with all of the dependencies installed
 
     ```sh
     conda config --add channels conda-forge
-    conda create -n build -y cmake boost-cpp libtiff fftw ninja
+    conda create -n build -y cmake libboost-devel libtiff fftw ninja cuda-nvcc libcufft-dev

```

Comment:
Note this only works for CUDA 12. That said, this is a reasonable choice from a docs perspective

Reply:
yeah, thank you.  i definitely used those locally and missed it here

Reply:
oh, sorry :joy: yes, you're pointing out that it's cuda12 specific.  Yeah, i could include that ...

Reply:
Yeah

Though starting developers off with the newer thing seems more reasonable and should be easier to get started with